scripts/ci/build_x86_64.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+# CC LD AR CXX has to be predifubed
+#
+
+export PKG_CONFIG_PATH="$HOME/cunit-install/x86_64/lib/pkgconfig:${PKG_CONFIG_PATH}"
+
+CWD=$(dirname "$0")
+TDIR=`mktemp -d -p ~`
+
+cd ${TDIR}
+export CROSS_ARCH=""
+
+export TARGET="x86_64$DPDKCC"
+
+$CWD/build_dpdk.sh
+DPDKPATH=`cat /tmp/dpdk_install_dir`
+
+echo 1000 | tee /proc/sys/vm/nr_hugepages
+mkdir -p /mnt/huge
+mount -t hugetlbfs nodev /mnt/huge
+
+git clone ${CWD}/../../ odp
+cd ./odp
+./bootstrap
+./configure ${CONF} \
+	--with-dpdk-path=${DPDKPATH}
+
+make clean
+make -j 8
+# Tell some time sensative ODP test that they can be skipped due to not
```

Comment:
sensitive

Reply:
Also docker should be pretty much isolated. I'd suggest commenting instead that we ignore possible failures there because these tests depends on measurements and systems might differ in performance.

Reply:
ok